src/redux/actions.ts

```diff
@@ -50,22 +52,25 @@ function signalAction<T extends ActionName> (t: T): Save<T, {}> {
   }
 }
 
-function makeAction<A extends Object, T extends ActionName> (t: T): ((formData: A) => Save<T, A>) {
+function makeAction<A extends Object, T extends ActionName> (t: T, validate: ValidateFunction<A>): ((formData: A) => Save<T, A>) {
```

Comment:
Can makeAction and makeAction2 be renamed to be more descriptive

Reply:
better?

Reply:
Looks great!